### PR TITLE
add ai server deploy

### PR DESCRIPTION
**Pull request by contfire**

### CODE REVIEW

File: ai/server deploy/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - /ml/model:/model
+    image: aichat:latest
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+    entrypoint: ["python3.9", "-m", "fastchat.serve.model_worker", "--model-names", "${MODEL_NAMES}", "--model-path", "${MODEL_PATH}", "--worker-address", "http://model-worker:21002", "--controller-address", "http://controller:21001", "--host", "0.0.0.0", "--port", "21002"]
```
Comment (XinShuYang) [inline ai/server deploy/docker-compose.yml]: Could you also provide another docker-compose.yml to support fast chat cli mode?

File: ai/github_issue_handling/issue_clawer.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
```
Comment (XinShuYang) [inline ai/github_issue_handling/issue_clawer.py:2]: This PR is only for server deployment. You may remove other files which exist in https://github.com/XinShuYang/antrea/pull/5/files .

**Comment by XinShuYang** [comment]
Please merge multiple commits into one.

**Comment by XinShuYang** [comment]
Do training and deployment models share the same Dockerfile?

File: ai/server deploy/docker-compose.yml (hunk 1 above)
Comment (contfire) [reply ai/server deploy/docker-compose.yml]: Updated！

**Comment by contfire** [comment]
Add the model debug cli and  refresh the server commit info.

**Comment by contfire** [comment]
Add automation training files

File: ai/github_issue_handling/issue_clawer.py (hunk 2 above)
Comment (XinShuYang) [reply ai/github_issue_handling/issue_clawer.py:2]: Reminder

File: ai_model/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+version: "3.9"
```
Comment (XinShuYang) [inline ai_model/docker-compose.yml:1]: Create another deployment folder for it?

**Pull request merged**